components/math/multiplication.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from "react";
+import { randomize } from "../../pages/api/questions/questionGenerator";
+import { getRandomItemFromArray } from "../../pages/api/random";
+
+export function getRndBlock() {
```

Comment:
Let's not do the randomization now. Let's just create a component that's only job is to render a square given some string property